common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -239,7 +245,10 @@ def get_html(self):
             'element_class': self.location.category,
             'ajax_url': self.system.ajax_url,
             'num_inputs': self.num_inputs,
-            'submitted': self.submitted
+            'submitted': self.submitted,
+            'instructions_default': WordCloudFields.instructions.default,
+            'instructions': self.instructions,
+            'display_name': self.display_name
```

Comment:
@clrux No, it's not how we handle it in other components. But in other components, the display name was always shown in the LMS.

My concern here (which I believe is @sstack22's concern as well) as that Word Cloud instances in running courses will suddenly show the default Word Cloud display name, as course teams will not have thought to set it previously.